app/services/user_updater.rb

```diff
@@ -65,13 +65,13 @@ def update(attributes = {})
       user_profile.website = format_url(attributes.fetch(:website) { user_profile.website })
     end
 
-    if attributes[:profile_background_upload_url] == ""
+    if attributes[:profile_background_upload_url] == "" || !user.has_trust_level?(SiteSetting.min_trust_level_to_allow_profile_background.to_i)
```

Comment:
These should be calls to the guardian methods. We want to keep the authorization logic in one place.

Reply:
said changes done @eviltrout